code/datums/progressbar.dm

```diff
@@ -21,7 +21,7 @@
 /datum/progressbar/New(mob/User, goal_number, atom/target)
 	. = ..()
 	if (!istype(target))
-		EXCEPTION("Invalid target given")
+		throw EXCEPTION("Invalid target given")
```

Comment:
Nothing is setup to catch this, it should be a stacktrace + qdel rather then an exception throw
```suggestion
		stack_trace("Invalid target [target] passed in")
		qdel(src)
		return
```

Reply:
world/Error automatically catches it, and it's automatically placed into the runtime viewer. This pr was infact an excuse to share that knowledge.

Reply:
damn you

Reply:
L